jack.py

```diff
@@ -1533,6 +1562,65 @@ def get_array(self):
         return np.frombuffer(self.get_buffer(), dtype=np.float32)
 
 
+def _maybe_bytes(data):
+    if type(data) is bytes:
+        data_ptr = _ffi.new("unsigned char[]", len(data))
+        data_ptr[0:len(data)] = data
+        return data_ptr
+    return data
+
+def _returns_bytes(fun):
+    def wrapper(self, cnt):
+        ret_cnt, data = fun(self, cnt)
+        return bytes(_ffi.buffer(data, ret_cnt))
+    return wrapper
+
+class RingBuffer(object):
+    def __init__(self, size=256):
+        ptr = _lib.jack_ringbuffer_create(size + 1)
+        if not ptr:
+            raise JackError(
+                "Could not create ringbuffer of size {0}".format(size))
```

Comment:
I would remove the size information from the message.
I think `"Could not create ringbuffer"` is enough.